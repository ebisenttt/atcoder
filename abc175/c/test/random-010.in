-742914975262566 954399026282610 74038845069300
