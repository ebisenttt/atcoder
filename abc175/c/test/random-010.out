2526524569566
